apps/myaccount/src/components/multi-factor-authentication/authenticators/fido-authenticator.tsx

```diff
@@ -495,7 +494,14 @@ export const FIDOAuthenticator: React.FunctionComponent<FIDOAuthenticatorProps>
                         <Grid.Column width={ 12 } className="first-column">
                             <List.Content>
                                 <List.Header>{ t("myAccount:components.mfa.fido.heading") }</List.Header>
-                                <List.Description>{ t("myAccount:components.mfa.fido.description") }</List.Description>
+                                <List.Description style={ { marginTop: "10px" } }>
```

Comment:
Better if we can move this to CSS files or use `mt`.

Reply:
Fixed